File: 1469-minimum-number-of-steps-to-make-two-strings-anagram/minimum-number-of-steps-to-make-two-strings-anagram.py
class Solution:
    def minSteps(self, s: str, t: str) -> int:
        countS = [0] * 26
        countT = [0] * 26

        for char in  s:
            countS[ord(char) - ord('a')] += 1
        for char in t:
            countT[ord(char) - ord('a')] += 1
        
        steps = 0
        for i in range(26):
            steps += abs(countS[i] - countT[i])
        
        return steps // 2

File: 1469-minimum-number-of-steps-to-make-two-strings-anagram/minimum-number-of-steps-to-make-two-strings-anagram.java
class Solution {
    public int minSteps(String s, String t) {
        int[] countStringOne = new int[26];
        int[] countStringTwo = new int[26];

        for (int i = 0; i < s.length(); i++) {
            countStringOne[s.charAt(i) - 'a']++;
            countStringTwo[t.charAt(i) - 'a']++;
        }

        int stepsRequired = 0;
        for (int i = 0; i < 26; i++) {
            stepsRequired += Math.abs(countStringOne[i] - countStringTwo[i]);
        }

        return stepsRequired / 2;
    }
}
